src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+  const Person = {
+    name: name,
+    age: age,
+  }
+    return Person;
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
This works but could be simplified quite a bit:
```suggestion
  return person.age > 65;
```